source/custom/nixnet_service.custom.cpp

```diff
@@ -1083,12 +1121,13 @@ ::grpc::Status NiXnetService::DbGetDatabaseList(::grpc::ServerContext* context,
     u32 number_of_databases{};
 
     status = library_->DbGetDatabaseList(ip_address, size_of_alias_buffer, const_cast<char*>(alias_buffer.c_str()), size_of_file_path_buffer, const_cast<char*>(file_path_buffer.c_str()), &number_of_databases);
-    response->set_status(status);
-    if (status_ok(status)) {
-      response->set_alias_buffer(alias_buffer.c_str());
-      response->set_filepath_buffer(file_path_buffer.c_str());
-      response->set_number_of_databases(number_of_databases);
+    if (!status_ok(status)) {
```

Comment:
Update the status check on line 1114 for `library_->DbGetDatabaseListSizes`.

Reply:
Argh!!!